backend/molgenis-emx2-rdf/src/main/java/org/molgenis/emx2/rdf/RDFService.java

```diff
@@ -423,6 +462,11 @@ public void rowsToRdf(final ModelBuilder builder, Table table, final String rowI
       } else {
         builder.add(subject, RDF.TYPE, tableIRI);
         builder.add(subject, RDF.TYPE, IRI_OBSERVATION);
+        if (table.getMetadata().getSemantics() != null) {
```

Comment:
I would say that if the table semantics are set that we would use that instead of the above triples, not in addition to these.